src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import model.Figure;
+import utility.FigureSupplier;
+
+public class Application {
+    private static final int MAX_LENGTH_ARRAY = 6;
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[MAX_LENGTH_ARRAY];
+
+        for (int i = 0; i <= figures.length - 1; i++) {
```

Comment:
The loop condition should be `i < figures.length` instead of `i <= figures.length - 1`. It's more conventional and easier to read.